src/lib/draw/route/GeocoderControls.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import type { FeatureCollection } from "@maptiler/geocoding-control/types";
+  import type { Position } from "geojson";
+  import { TextInput } from "lib/govuk";
+  import { emptyGeojson } from "lib/maplibre";
+  import { map, routeTool } from "stores";
+  import { geocoderGj } from "./stores";
+
+  let query = "";
+
+  async function runQuery(query: string) {
+    if (!query) {
+      $geocoderGj = emptyGeojson() as FeatureCollection;
+      return;
+    }
+
+    let params = new URLSearchParams({
+      country: "gb",
+      proximity: $map.getCenter().toArray().join(","),
+      fuzzyMatch: "true",
+      key: import.meta.env.VITE_MAPTILER_API_KEY,
+    }).toString();
+    let url = `https://api.maptiler.com/geocoding/${query}.json?${params}`;
```

Comment:
https://docs.maptiler.com/cloud/api/geocoding/ for reference. Maybe we should specify the bbox as the authority boundary area and further limit results?